### PR TITLE
Show map city

### PR DESCRIPTION
**Pull request by akopsykokwak**

### CODE REVIEW

File: src/App.jsx [hunk 1]
```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+// import InitMap from './Components/InitMap';
```
Comment (TonicVi) [inline src/App.jsx]: line to delete, no comments on pull requests

File: src/components/InitMap.jsx [hunk 2]
```diff
@@ -0,0 +1,43 @@
+import React, { components } from 'react';
+import { Map, CircleMarker, TileLayer } from 'react-leaflet';
+import './InitMap.css';
+import 'leaflet/dist/leaflet.css';
+import axios from 'axios';
+
+class InitMap extends React.Component {
```
Comment (TonicVi) [inline src/components/InitMap.jsx:7]: if Componenent already imported, you shouldn't extend 'React.Componenent'

File: src/components/InitMap.jsx [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+  constructor(props) {
+    super(props);
+    // eslint-disable-next-line react/no-unused-state
```
Comment (TonicVi) [inline src/components/InitMap.jsx:10]: no comment

File: src/components/MapTest.jsx [hunk 4]
```diff
@@ -0,0 +1,38 @@
+import React, { components } from 'react';
+import { Map, CircleMarker, TileLayer, Tooltip } from 'react-leaflet';
+import 'leaflet/dist/leaflet.css';
+import './MapTest.css';
+import data from './data';
+
+// eslint-disable-next-line react/prefer-stateless-function
```
Comment (TonicVi) [inline src/components/MapTest.jsx:7]: no comment

Comment (TonicVi) [inline src/components/MapTest.jsx:7]: no comments

File: src/components/InitMap.jsx (hunk 3 above)
Comment (TonicVi) [inline src/components/InitMap.jsx:10]: no comments

File: src/components/SearchBar.jsx [hunk 5]
```diff
@@ -0,0 +1,49 @@
...
+    const loweredCoordinates = coordinates.toLowerCase();
+    this.props.dataInput(loweredCoordinates);
+  }
+
+  /* Laisse le texte affiché dans la search bar */
+  // componentDidMount() {
+  //   // eslint-disable-next-line no-undef
+  //   const coordinates = localStorage.getItem('coordinates');
+  //   this.setState({ coordinates })
+  // }
```
Comment (TonicVi) [inline src/components/SearchBar.jsx]: no comments

**Review by TonicVi — APPROVED** [review]

File: src/components/MapTest.jsx [hunk 6]
```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import { Map, CircleMarker, TileLayer, Tooltip } from 'react-leaflet';
+import 'leaflet/dist/leaflet.css';
+import './MapTest.css';
+import data from './data';
+
```
Comment (Froyj) [inline src/components/MapTest.jsx:6]: Boooooh

**Review by Froyj — APPROVED** [review]
ack !

**Pull request merged**